operator/pkg/ciliumidentity/pod.go

```diff
@@ -16,3 +21,14 @@ type PodItem struct {
 func (p PodItem) Key() resource.Key {
 	return p.key
 }
+
+func (c *Controller) processPodEvents(ctx context.Context) error {
+	for event := range c.pod.Events(ctx) {
+		if event.Kind == resource.Upsert || event.Kind == resource.Delete {
+			c.logger.Debug("Got Pod event", logfields.Type, event.Kind, logfields.K8sPodName, event.Key.String())
```

Comment:
Yeah, you're right, didn't notice that. Although the perf impact should be minimal (especially because most of the logging for operator identity mgmt is at debug level) better be consistent across the codebase. :+1: